infrastructure/io/src/main/java/tech/pegasys/teku/infrastructure/io/resource/FileResourceLoader.java

```diff
@@ -28,10 +29,14 @@ protected FileResourceLoader(final Predicate<String> sourceFilter) {
 
   @Override
   Optional<InputStream> loadSource(final String source) throws IOException {
-    final Path path = Path.of(source);
-    if (!path.toFile().exists()) {
+    try {
+      final Path path = Path.of(source);
+      if (!path.toFile().exists()) {
+        return Optional.empty();
+      }
+      return Optional.of(Files.newInputStream(path));
+    } catch (InvalidPathException e) {
```

Comment:
hmm, interesting.  I wonder if we should just be writing this in terms of `File` rather than `Path`. Does the code below work for you:

```
  Optional<InputStream> loadSource(final String source) throws IOException {
    final File file = new File(source);
    if (!file.exists()) {
      return Optional.empty();
    }
    return Optional.of(new BufferedInputStream(new FileInputStream(file)));
  }
```

Reply:
btw I think I'd prefer the file based option I posted if it works, but would be ok with either of these fixes being merged into Teku master.

Reply:
@ajsutton thanks for suggested fix: it works fine 👍 
Opened PR to Teku master: https://github.com/ConsenSys/teku/pull/4360